gpt_index/readers/file.py

```diff
@@ -150,6 +175,8 @@ def __init__(
         num_files_limit: Optional[int] = None,
         file_metadata: Optional[Callable[[str], Dict]] = None,
         verbose: bool = False,
+        video_audio_files: bool = False,
```

Comment:
Understood. 
`data = self.file_extractor[input_file.suffix](input_file, self.errors, DEFAULT_FILE_EXTRACTOR_CONFIGS)`

do you mean passing this way?
Also, the whisper "large-v1" model gives better results, so it would be good to have it as a parameter for users to experiment with. What do you think?